app/views/users/index.html.erb

```diff
@@ -1 +1,8 @@
-<h1>index for users</h1>
\ No newline at end of file
+<% @users.each do |user|%>
+<div>
+  <img src='#{ <%= user.photo %> }' width='100' height='100' />
+  <span><%= user.name%></span>
+  <p>number of posts: <%=user.posts_counter || 0%></p>
+  <hr />
```

Comment:
- [ ] By Clicking on each user must take you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)Clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png). Kindly make this 👍🏻

Reply:
done